index.html

```diff
@@ -0,0 +1,417 @@
+      <div class="view-price__blur-gradient"></div>
+    </section>
+  </main>
+  <footer class="footer">
+    <ul class="footer__list">
+      <li class="footer__list-item">
+        <ul class="footer__link-list">
+          <li class="footer__link-list-item">
+            <h5 class="heading heading--sm heading--light">Company</h5>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">About Us</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Why Choose us</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Pricing</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="" class="link link--light">Testimonial</a>
+          </li>
+        </ul>
+      </li>
+      <li class="footer__list-item">
+        <ul class="footer__link-list">
+          <li class="footer__link-list-item">
+            <h5 class="heading heading--sm heading--light">Resources</h5>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Privacy Policy</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Terms and Conditionx</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Blog</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="" class="link link--light">Contact Us</a>
+          </li>
+        </ul>
+      </li>
+      <li class="footer__list-item">
+        <ul class="footer__link-list">
+          <li class="footer__link-list-item">
+            <h5 class="heading heading--sm heading--light">Product</h5>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Project managment</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Terms and Conditionx</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="#" class="link link--light">Time tracker</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="" class="link link--light">Time schedule</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="" class="link link--light">Lead generate</a>
+          </li>
+          <li class="footer__link-list-item">
+            <a href="" class="link link--light">Remote Collaboration</a>
+          </li>
+        </ul>
+      </li>
+      <li class="footer__list-item">
+        <div class="footer__logo">
+          <img src="./assets/white-logo.svg" alt="logo" class="logo__img">
+        </div>
+        <div class="footer__newsletter">
+          <p class="text text--bold text--light">
```

Comment:
Here also, you should write footer__title footer__title--bold footer__title--light